src/QueueableCommand.php

```diff
@@ -3,12 +3,11 @@
 namespace League\Tactician\Bernard;
 
 use Bernard\Message;
-use League\Tactician\Command;
 
 /**
  * Creates a common interface for queueable commands
  */
-interface QueueableCommand extends Command, Message
+interface QueueableCommand extends Message
```

Comment:
Yeah, had the same thought, wanted to talk it over with you. On the one hand, I like it, on the other hand if we ever want to change it or add something else on, we kinda can't come back from this. (That said, I think the future of this type of configuration is probably some sorta per command metadata store but that's way beyond the scope of this ticket and more of a 2.0 for everything). So, think it over, but I'm okay with it personally.